### DISCUSSION TITLE
How to close horizontal terminals?

### DISCUSSION THREAD

**Discussion by ghost** [NvChad/NvChad#1038] [Q&A] [answered]
My custom configuration is not being loaded and now I'm creating horizontal terminals by mistake. How do I close them?

**Comment by siduck** [comment, marked as answer]
leader + x

**Comment by ericwq** [comment]
:exit

**Comment by Applicafroguy** [comment]
first `Ctr + x` to exit and then `leader + x` to close

**Reply by siduck** [reply]
@Applicafroguy that wont close but hide it, so just type exist in terminal
